widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -1374,6 +1379,49 @@ div#FilterPanel div.pentaho-toggle-button-up button {
 .flex-wrap {
   flex-wrap: wrap;
 }
+
+.flex-center {
+  justify-content: center;
+  align-items: center;
+}
+
+:is(.flex-column.flex-center-v, .flex-row.flex-center-h) {
+  justify-content: center;
+}
+
+:is(.flex-row.flex-center-v, .flex-column.flex-center-h) {
+  align-items: center;
+}
+
+/* Align the content at bottom, when using the flex-row */
+.flex-row.flex-bottom {
+  align-items: flex-end;
+}
+
+/* Align the content at bottom, when using the flex-column */
+.flex-column.flex-bottom {
+  justify-content: flex-end;
+}
+
+@media (max-width: 640px) {
+  :where(.flex-row).flex-column-sm {
+    flex-direction: column;
+  }
+
+  :where(.flex-row).flex-column-sm.flex-center-v {
+    justify-content: center;
+    align-items: unset;
+  }
+
+  :where(.flex-row).flex-column-sm.flex-center-h {
+    align-items: center;
+    justify-content: unset;
+  }
+
+  :where(.flex-row) .picklistPanel {
+    transform: rotate(90deg);
+  }
```

Comment:
This rule should be together with other rules of the same dialog. Unfortunately, that means repeating the media query statement in other places.